platform/android/display_server_android.cpp

```diff
@@ -392,6 +392,19 @@ void DisplayServerAndroid::reset_window() {
 #endif
 }
 
+void DisplayServerAndroid::notify_surface_changed(int width, int height) {
```

Comment:
To match the style, can you rename the method parameters to `p_width` and `p_height`.

Reply:
Done.